rules/windows/persistence_run_key_and_startup_broad.toml

```
@@ -17,6 +17,66 @@ index = ["logs-endpoint.events.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Startup or Run Key Registry Modification"
+note = """## Triage and analysis
+
+### Investigating Startup or Run Key Registry Modification
+
+Adversaries may achieve persistence by referencing a program with a Registry run key. Adding an entry to the run keys
+in the Registry will cause the program referenced to be executed when a user logs in. These programs will executed
+under the context of the user and will have the account's permissions. This rule looks for this behavior by monitoring
+a range of registry run keys.
```

I think the full proper name "Windows Registry" is capitalized, but lowercase here for just generally referring to the registry:
```suggestion
Adversaries may achieve persistence by referencing a program with a registry run key. Adding an entry to the run keys
in the registry will cause the program referenced to be executed when a user logs in. These programs will executed
under the context of the user and will have the account's permissions. This rule looks for this behavior by monitoring
a range of registry run keys.
```